compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_method_gen.bal

```diff
@@ -875,7 +939,7 @@ function castFromString(bir:BType targetType, jvm:MethodVisitor mv) {
 # 
 # + targetType - target type to be casted
 # + mv - method visitor
-function checkCastFromObject(bir:BType targetType, jvm:MethodVisitor mv) {
+function checkCastFromObject(bir:BType? targetType, jvm:MethodVisitor mv) {
```

Comment:
Not related to this PR, but can we use the `addBoxInsn(...)` method instead of  `generateObjectCast()`? These two functions looks duplicate to me. I think we should remove one of those. 

Same for using `addUnboxInsn()` instead of `checkCastFromObject()`

Reply:
+1, will fix in a subsequent PR.